java/com/jfreechartextensions/axistransformations/ExtendedNumberAxis.java

```diff
@@ -324,12 +346,19 @@ private NumberTick createTick(final TickType tickType, final RectangleEdge edge,
         }
     }
 
-    private String formatLabel(final double tickValue){
+    private String formatLabel(final double tickValue) {
         final NumberFormat formatter = getNumberFormatOverride();
         return formatter == null ? getTickUnit().valueToString(tickValue) : formatter.format(tickValue);
     }
 
-    private boolean okToPlotTick(final double tickValue){
+    /**
+     * Checks whether the tickValue is valid to draw
+     *
+     * @param tickValue - tick value to check
+     * @param isMajor   - whethe tick is major or minor
+     * @return boolean indicating whether the tick value is valid to show
+     */
+    protected boolean okToPlotTick(final double tickValue, final boolean isMajor) {
```

Comment:
What does this isMajor do? Doesn't seem used

Reply:
It basically shows whether the tick is major or minor. It is used in iris project. In iris, for DateTimeAxis with BusinessCalendar transformation, if we the tick is major, we're avoiding it and just processing minor ticks.